mcu_program/common_src/lms7002m_filters.c

```diff
@@ -170,12 +170,14 @@ uint8_t TuneRxFilterSetup(const float_type rx_lpf_IF)
 */
 #endif
     Modify_SPI_Reg_bits(G_TIA_RFE, g_tia_rfe);
-    if(g_pga_rbb == 31)
+    if(g_pga_rbb == 31) {
         Modify_SPI_Reg_bits(G_PGA_RBB, 22);
-    else
+    }
+    else {
         Modify_SPI_Reg_bits(G_PGA_RBB, g_pga_rbb);
+    }
 
-	{
+    {
```

Comment:
This indentation and brackets was definitely unclear. just to double check, @IgnasJarusevicius was this the original intention?